Fluid/Filters/MiscFilters.cs

```diff
@@ -264,10 +265,27 @@ public static ValueTask<FluidValue> ChangeTimeZone(FluidValue input, FilterArgum
 
             var timeZone = arguments.At(0).ToStringValue();
 
-            if (!TZConvert.TryGetTimeZoneInfo(timeZone, out var timeZoneInfo)) return new DateTimeValue(value);
+
+#if HAS_TIMEZONE_API
```

Comment:
What about the new NET_6_OR_GREATER kind of directives?

Reply:
It would be much better

Reply:
Good and bad at the same time. When I created it I thought there was a new API to do it, but actually it's just a new behavior of the existing API. So yes in this case `NET_6_OR_GREATER` will be better. However in general it's better to use custom values like this.